examples/skr/TROUBLESHOOTING.md

```diff
@@ -53,3 +53,57 @@ err: AKV post request failed: HTTP GET failed: Post "https://<mhsm-name>.managed
 Ensure that:
 
 - the name of the mHSM is correct in the ARM template and matches the name of the mHSM in the importkeyconfig.json file
+
+## "CreateContainerRequest is blocked by policy" error (Confidential Containers AKS Only)
+
+If you are running the secure key release container on confidential containers AKS and the container pod does not run. Occasionally this could be caused by incorrect workload identity setup that results in empty environment variables that the container depends on. To debug, issue the following command: 
+
+```bash
+# This command describes the current status of the container pod
+kubectl describe pod skr 
+```
+If the error is the following: 
+
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim task: "CreateContainerRequest is blocked by policy": unknown
```

Comment:
I would put the ``` ``` around the error message, just to be consistent with the formatting of the rest of the doc.